packages/ui-text-input/src/TextInput/README.md

```diff
@@ -301,6 +298,47 @@ Focusable content will be focused separately from the input itself.
   render(<ExtraContentExample />)
   ```
 
+Another common usecase is to add an `IconButton` at the end of a TextInput, e.g. for revealing the content of a password field. In these cases, please use the `withBorder={false}` and `withBackground={false}` props for the IconButton.
+
+```js
+---
+type: example
+---
+const InputsWithButtonsExample = () => {
+  const [passwordValue, setPasswordValue] = useState('')
+  const [showPassword, setShowPassword] = useState(false)
+  return (
+    <TextInput
+      renderLabel="Password"
+      type={showPassword ? 'text' : 'password'}
+      placeholder="Find something..."
+      value={passwordValue}
+      onChange={(e, newValue) => setPasswordValue(newValue)}
+      renderAfterInput={
+        <IconButton withBorder={false} withBackground={false} onClick={() => setShowPassword(prevState => !prevState)}>
+          {showPassword ? <IconOffLine/> : <IconEyeLine/>}
+        </IconButton>
```

Comment:
SRs say here just Button, please add some text for them, something like "show/hide text"